src/plugins/cosmos/components/modals/Send/views/QrCodeScanner.tsx

```diff
@@ -0,0 +1,58 @@
+import { Button, ModalBody, ModalCloseButton, ModalFooter, ModalHeader } from '@chakra-ui/react'
+import { lazy, Suspense } from 'react'
+import { useFormContext } from 'react-hook-form'
+import { useTranslate } from 'react-polyglot'
+import { useHistory } from 'react-router-dom'
+import { SlideTransition } from 'components/SlideTransition'
+import { Text } from 'components/Text'
+
+import { SendFormFields, SendInput } from 'components/Modals/Send/Form'
+import { SendRoutes } from '../Send'
+
+const QrReader = lazy(() => import('react-qr-reader'))
+
+export const QrCodeScanner = () => {
+  const history = useHistory()
+  const translate = useTranslate()
+  const { setValue } = useFormContext<SendInput>()
+
+  const handleError = () => {
+    /** @todo render error to user */
```

Comment:
https://github.com/shapeshift/web/blob/a24254f45b7aabe81a8a09ef7f13ce03ada5c23f/src/components/Modals/Send/views/QrCodeScanner.tsx

I implemented this some weeks ago, you'll need to handle these errors

I think it would be worth to extract this logic into a reusable components instead of duplicating the error handling, but I may be wrong